pkg/sbom/sbom.go

```diff
@@ -64,7 +64,7 @@ func Init(e workflow.Engine) error {
 
 func newData(contentType string, sbom []byte) workflow.Data {
 	return workflow.NewData(
-		WorkflowID,
+		workflow.NewTypeIdentifier(WorkflowID, "cyclonedx"),
```

Comment:
@PeterSchafer would be cool if we could use `cyclonedx+json`, since we will also add support for XML flavour at some point.

Reply:
There are the following types:
* **TypeIdentifier** is similar to a class/struct name (cyclonedx)
* **Mimetype** is the type of the data representation (application/vnd.cyclonedx+json)

Does this make sense?

Reply:
i’m not sure i understand yet. isn’t the type identifier directly bound to the data contents, i.e. shouldn’t payloads `[]byte("{\"blah\":\"blub\",...}")` and `[]byte("<?xml version ...>")` receive different identifiers, even if they were both to be valid CycloneDX?